store/src/java/com/zimbra/qa/unittest/TestScheduledTaskManager.java

```diff
@@ -29,115 +35,117 @@
 import com.zimbra.cs.mailbox.ScheduledTask;
 import com.zimbra.cs.mailbox.ScheduledTaskManager;
 
-public class TestScheduledTaskManager
-extends TestCase {
+public class TestScheduledTaskManager {
+    @Rule
+    public static TestName testInfo = new TestName();
 
     static final String TASK_NAME = "TestTask";
-    private static final String USER_NAME = "user1";
-    
-    public void setUp()
-    throws Exception {
-        cleanUp();
+    private static final String USER_NAME = String.format("%s-%s-user1", TestScheduledTaskManager.class.getSimpleName(), testInfo.getMethodName()).toLowerCase();
```

Comment:
this does not break the test, but as in the other PR, getMethodName() will probably return an empty string in this context.